libbeat/scripts/generate_fields_docs.py

```diff
@@ -62,6 +62,10 @@ def document_field(output, field, path):
         if not field["enable"]:
             output.write("{}\n\n".format("Object is not enabled."))
 
+    if "multi_fields" in field:
+        for subfield in field["multi_fields"]:
+            document_field(output, subfield, path + "." + subfield["name"])
+
```

Comment:
good catch.

Reply:
@simitt Mentioning you here as apm-server uses multi fields as far as I remember. Not really a bug but just in case you are surprised why the docs are update with the most recent beats version.